### PR TITLE
Make pull request template

### PR DESCRIPTION
**Pull request by klaudia-klis**
## Click Up Task Link
[Build a template for PR](https://app.clickup.com/t/86930g31h)

## What has been done
Build a template for pull requests.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## Click Up Task Link
+[Paste here task description from ClickUp](Paste here link to ClickUp task)
+
+## What has been done
```
Comment (choeqq) [inline .github/pull_request_template.md]:
```suggestion
## Click Up Task Link ⏰ 
[Paste here task description from ClickUp](Paste here link to ClickUp task)

## What has been done 😄 
```

**Review by choeqq — APPROVED** [review]
Looks great, I'd just add some emojis just to make it fun 😄

**Comment by klaudia-klis** [comment]
I think to make this work I should create this file on the main branch, without creating pr-template branch

**Comment by choeqq** [comment]
Just merge it if possible I think it should easily work 😄 
You need to request approve from one of team members to merge it ✅

**Review by choeqq — APPROVED** [review]

**Review by PawelP2001 — APPROVED** [review]

**Review by Vira-V — APPROVED** [review]

**Pull request merged**